tests/test_stitched_psf.py

```diff
@@ -0,0 +1,247 @@
+# This file is part of cell_coadds.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import pickle
+import unittest
+
+import lsst.geom as geom
+import lsst.meas.base.tests
+import lsst.utils.tests
+import numpy as np
+from lsst.afw.detection import GaussianPsf
+from lsst.afw.image import ImageD
+from lsst.cell_coadds import GridContainer, StitchedPsf, UniformGrid
+from lsst.meas.algorithms import SingleGaussianPsf
+from lsst.skymap import Index2D
+
+
+class StitchedPsfTestCase(lsst.utils.tests.TestCase):
+    """Test the methods of StitchedPsf class."""
```

Comment:
Much of these tests redo what is already being done in `test_coadds.py`, but they should really live here. I left the original ones in `test_coadd.py` though, and I don't think that that adds to the build times in any significant manner.